docs/blog/articles/2023-10-08-KServe-0.11-release.md

```diff
@@ -0,0 +1,143 @@
+# Announcing: KServe v0.11
+
+We are excited to announce the release of KServe 0.11, in this release we introduced LLM runtimes, made enhancements to the KServe control plane, Python SDK Open Inference Protocol support and dependency managemenet. 
```

Comment:
most people would probably know it already, but should we spell out "Large Language Model (LLM)" at least once?